tests/system/source/main.d

```diff
@@ -25,42 +25,28 @@ import agora.utils.PrettyPrinter;
 import vibe.web.rest;
 
 import std.algorithm;
+import std.exception;
 import std.stdio;
 import core.thread;
 import core.time;
 
-/// Helper struct
-private struct Address
-{
-    ///
-    string host;
-    ///
-    ushort port;
 
-    /// Helper function to call make the address vibe.inet.URL friendly
-    public string withSchema () const @safe
+void main (string[] args)
+{
+    if (args.length < 2)
     {
-        import std.format;
-        return format("http://%s:%d", this.host, this.port);
+        writeln("You must enter addresses of the nodes to connect to.");
+        writeln("   ex) http://127.0.0.1:4000 http://127.0.0.1:4001 http://127.0.0.1:4002 ...");
+        return;
```

Comment:
This return with a success code. That's not good, because then any script, and the CI, will see it as successful. Return 1.